src/ravenpy/config/emulators/__init__.py

```diff
@@ -9,26 +9,18 @@
 from ravenpy.config.emulators.sacsma import SACSMA
 
 __all__ = [
-    "GR4JCN",
-    "HBVEC",
-    "HMETS",
-    "HYPR",
-    "SACSMA",
-    "BasicRoute",
-    "Blended",
-    "CanadianShield",
-    "Mohyse",
```

Comment:
That's not just a doc change, it's going to affect how people import emulators, no ?

Reply:
They still technically work, since they're imported above. I added these into _all__ in the last two PRs, so we could think of this as a correcting a regression I introduced. The notebooks will still work exactly as intended.

I think the complicating factor is that we can't easily advertise these models in the docs without creating JSON representations of them (which means generating all their subcomponents).

The [stable](https://ravenpy.readthedocs.io/en/stable/user_api.html#module-ravenpy.config.emulators) build (last release) of the documentation predates all this and doesn't list these models either.